### DISCUSSION TITLE
Accessing tracks and photos following Android 11 storage policy change

### DISCUSSION THREAD

**Discussion by tommycrock** [osmandapp/OsmAnd#12944] [Q&A]
Hi there. I understand that

> Starting with the November updates, OsmAnd will no longer be able to support Shared storage. The other options will still be available. https://groups.google.com/g/osmand/c/7mWxB_tM_xA/m/k3gqXOtiAwAJ

Does this mean a custom path will still be available, allowing us to type /storage/emulated/0/osmand, or is this also disallowed? 

If we can't use shared storage how can we access tracks, photos, etc.

Thanks. I realise osmand didn't create this issue!

**Comment by vshcherb** [comment]
Some phones may provide access and some not. Backup + Share functionality will still be available

**Reply by tommycrock** [reply]
Thanks for the reply. Does that mean custom storage location may be available on some phones or that access to the android folder may be available? Thanks

**Reply by vshcherb** [reply]
Yes, you can always specify path manually and check if it works

**Comment by Boothy99** [comment]
Old discussion, but if you've not found how to access the "hidden" files" have a look here -
https://github.com/osmandapp/OsmAnd/discussions/15840#discussioncomment-6078140
